mdgo/forcefield.py

```diff
@@ -64,18 +69,13 @@
         "spc": "water_spc.lmp",
         "spce": "water_spce.lmp",
         "tip3pew": "water_tip3p_ew.lmp",
+        "tip3pfb": "water_tip3p_fb.lmp",
+        "opc3": "water_opc3.lmp",
         "tip4p2005": "water_tip4p_2005.lmp",
         "tip4pew": "water_tip4p_ew.lmp",
+        "tip4pfb": "water_tip4p_fb.lmp",
+        "opc": "water_opc.lmp",
     },
-    "ion": {
-        "aq": ["default"],
-        "aqvist": ["default"],
-        "jj": ["default"],
-        "jensen_jorgensen": ["default"],
-        "jc": ["spce", "tip3p", "tip4pew"],
-        "joung_cheatham": ["spce", "tip3p", "tip4pew"],
-    },
-    "alias": {"aq": "aqvist", "jj": "jensen_jorgensen", "jc": "joung_cheatham"},
 }
 
```

Comment:
```suggestion
WATER_SIGMA: Final[dict] = {
    "spc": 3.16557,
    "spce": 3.16557,
    "tip3p": 3.1507,
    "tip3pew": 3.188,
    "tip3pfb": 3.178,
    "opc": 3.16655,
    "opc3": 3.17427,
    "tip4p": 3.1536,
    "tip4p2005": 3.1589,
    "tip4pew": 3.16435,
    "tip4pfb": 3.1655,
}
```

Reply:
This works for now. My only concern, long-term, is that we have now duplicated the SIGMA values here and in the respective water data files. In my experience, having data duplicated increases the risk of bugs or unintended behavior later on, because someone might update a value in one place but forget to change it in the other (or not realize it's there).

So longer term, I think it would be good to have a single source of truth for the water model parameters, but this is good for now!